pxl_scripts/px/net_flow/net_flow.pxl

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Pixie Labs, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+import px
+
+
+def net_flow_graph(start: str, ns: px.Namespace, from_entity_filter: str, to_entity_filter: str):
+    df = px.DataFrame('conn_stats', start_time=start)
+
+    # Filter on namespace.
+    df = df[df.ctx['namespace'] == ns]
+
+    # Filter for client side requests.
+    df = df[df['role'] == 1]
+
+    # Store the pod. Ideally this would be done after the aggregate,
+    # but that's not working right now.
+    df['pod'] = df.ctx['pod']
+
+    # Filter out any non k8s sources.
+    df = df[df['pod'] != '']
+
+    # Use aggregate to pick the latest sample for any given client-server pair.
+    # We do this by picking the max of the stats, since they are all counters.
+    df = df.groupby(['pod', 'upid', 'remote_addr']).agg(
+        bytes_sent=('bytes_sent', px.max),
+        bytes_recv=('bytes_recv', px.max),
+    )
+    df.bytes_total = df.bytes_sent + df.bytes_recv
```

Comment:
I would probably switch everything to use the `df.colname` syntax, looks cleaner to me